src/ethernet/CMakeLists.txt

```diff
@@ -80,7 +80,7 @@ endif()
 set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
 
 target_link_libraries(${PROJECT_NAME}
-    PRIVATE ${WINLIB} realsense2 realsense2-compression
+    PUBLIC ${WINLIB} realsense2 realsense2-compression
```

Comment:
Why was this needed?

Reply:
I checked the network device and the CMake there (pyrealsense2-net, I think) didn't depend on realsense2, therefore didn't get rsutils, either. This fixes it.

Reply:
OK for here but I think you should start thinking of alternatives ways to fix this kind of issues.
changing to `PUBLIC` is a really bad practice.